packages/desktop-client/src/components/accounts/Header.tsx

```
@@ -89,18 +178,18 @@ export function AccountHeader({
   const [menuOpen, setMenuOpen] = useState(false);
-  const searchInput = useRef(null);
+  const searchInput = useRef<HTMLInputElement>(null);
   const triggerRef = useRef(null);
   const splitsExpanded = useSplitsExpanded();
   const syncServerStatus = useSyncServerStatus();
   const isUsingServer = syncServerStatus !== 'no-server';
   const isServerOffline = syncServerStatus === 'offline';
   const [_, setExpandSplitsPref] = useLocalPref('expand-splits');
 
-  let canSync = account && account.account_id && isUsingServer;
+  let canSync = !!(account && account.account_id && isUsingServer);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)